src/scripting/scriptengine.cpp

```diff
@@ -41,12 +41,12 @@
 #include "scriptengine_lua.inl"
 
 namespace {
-    const std::string MainTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/main.hbs";
-    const std::string ScriptingTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/scripting.hbs";
-    const std::string HandlebarsFilename = "${OPENSPACE_DATA}/web/common/handlebars-v4.0.5.js";
-    const std::string JsFilename = "${OPENSPACE_DATA}/web/luascripting/script.js";
-    const std::string BootstrapFilename = "${OPENSPACE_DATA}/web/common/bootstrap.min.css";
-    const std::string CssFilename = "${OPENSPACE_DATA}/web/common/style.css";
+    const char* MainTemplateFilename = "${OPENSPACE_DATA}/web/luascripting/main.hbs";
```

Comment:
const std::string has some initialization that happens at application startup time, whereas const char* can be directly used.  Probably noone will be able to notice, but i think it is a good habit not to use std::string if not necessary